src/index.tsx

```diff
@@ -1,2 +1,6 @@
 export { extendedTheme } from "./theme"
+
 export * from "./components"
```

Comment:
if we're extracting these components here, we should move `src/components/index` exports here (or vice versa) so all exports are happening in the same spot